featurebyte/session/base_spark.py

```diff
@@ -268,7 +280,8 @@ def create_metadata_table_queries(self, current_migration_version: int) -> List[
                     CURRENT_TIMESTAMP() AS CREATED_AT
```

Comment:
This insert statement was run unconditionally every time when the working schema is outdated, causing a new row to be added incorrectly.